.circleci/config.yml

```diff
@@ -393,3 +385,16 @@ workflows:
       - test_instrumented:
           requires:
             - compile
+
+  release:
+    jobs:
+      - compile:
+          filters:
+            tags:
+              only: /^v((20)[0-9]{2})\.\d+\.\d+$/ # matches semvers like v1.2.3
```

Comment:
I usually go with `v*` but this is fine too. I expect Collect to still be going strong in 2099! 😅